vendor/archive/tar/LICENSE

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2009 The Go Authors. All rights reserved.
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are
-met:
```

Comment:
whoops; looks like you accidentally removed this vendor as well

Reply:
I seem to always have troubles with vndr :(

Reply:
fixed